### ISSUE TITLE
Keras grouped YatConv fails for groups greater than one

### ISSUE THREAD

**Issue by mlnomadpy** [azettaai/nmn#73]
## Summary

Keras `YatConv1D/2D/3D` fail whenever `groups > 1`.

## Reproduction

With the JAX backend, input channels 4, filters 4, and groups 2, all dimensions raise:

```text
rhs output feature dimension size must be a multiple of feature_group_count,
but 1 is not a multiple of 2
```

The patch-norm helper kernel has one output channel while `ops.conv` infers grouped convolution.

## Acceptance criteria

- [ ] Produce one norm channel per group and repeat within each group.
- [ ] Test 1D/2D/3D grouped forwards and gradients on JAX and TensorFlow backends.
